chef/data_bags/crowbar/migrate/network/101_add_ceph_client_network.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(ta, td, a, d)
+  a["networks"]["ceph_client"] = ta["networks"]["ceph_client"] unless a["networks"].key? "ceph_client"
```

Comment:
Can you split this line? It is only small nitpick, but we would like to have Hound checks green.